flexget/plugins/input/anidb_feed.py

```diff
@@ -0,0 +1,479 @@
+            anidb_name = self.get_value_via_regex('title', new_entry, r'^(.*?) -')
+            # fix and add to new_entry()
+
+            quality_string = ''
+            if resolution_string:
+                quality_string += resolution_string
+            if source_string:
+                quality_string += ' ' + source_string
+            if not quality_string.isspace():
+                try:
+                    quality = Quality(quality_string)
+                    new_entry['quality'] = quality
+                except Exception as ex:
+                    log.error('Could not get Quality from string: %s, error: %s' % (quality_string, ex))
+            # build the new 'title' mimic general scene naming convention
+            title_new = '%s ' % anidb_name
+            if is_movie is not None:
+                new_entry['movie_name'] = anidb_name
+            else:
+                new_entry['series_name'] = anidb_name
+            if 'ep' in episode_data:
+                new_entry['series_episode'] = episode_data['ep']
+                title_new += '- %s' % episode_data['ep']  # FIXME: is there a valid way to encode anime episodes?
+                new_entry['series_id'] = '%02d' % episode_data['ep']  # Is this valid aka a single Integer?
+                if episode_data['ep'] > 99:
+                    new_entry['series_id'] = '%03d' % episode_data['ep']
+            elif episode_data['extra'] is True:
+                extra_string = episode_data.get('special', '')
+                extra_string += episode_data.get('op-ending', '')
+                extra_string += episode_data.get('trailer-promo', '')
+                title_new += '- %s' % extra_string
+                new_entry['series_id'] = extra_string  # FIXME: is this the correct field for extras?
+            if episode_data.get('version'):
+                title_new += 'v%s' % episode_data['version']
+                new_entry[NAMESPACE_PREFIX_MAIN + 'fileversion'] = episode_data['version']  # TODO: Is there a official field?
+            if group_tag:
+                title_new += ' - [%s]' % group_tag  # TODO: do we always add group?
+                new_entry[NAMESPACE_PREFIX + 'grouptag'] = group_tag
+            if new_entry.get('quality') and hasattr(new_entry['quality'], 'resolution'):
+                if new_entry['quality'].resolution.name != 'unknown':
+                    title_new += ' [%s]' % new_entry['quality'].resolution
+
+            new_entry['title'] = title_new
+            new_entry[NAMESPACE_PREFIX_MAIN + 'name'] = anidb_name
+            entries.append(new_entry)
+            #_debug_dump_entry(new_entry)
+        return entries
+
+    def fill_entries_for_url(self, url, task, config):
+        log.verbose('Fetching %s' % url)
+
+        try:
+            r = task.requests.get(url, headers=HEADER, timeout=20)
+        except Exception as ex:
+            log.error("Failed fetching url: %s error: %s" % (url, ex))
+            return []
```

Comment:
Empty list can be ok return value sometimes ... but for failing to retrieve the thing we need? Usually we abort the task then.